web/components/feed/activity-items.ts

```diff
@@ -308,6 +311,37 @@ function groupBetsAndComments(
   return abbrItems
 }
 
+function getCommentsWithPositions(
+  bets: Bet[],
+  comments: Comment[],
+  contract: Contract
+) {
+  function mapBetsByUserId(bets: Bet[]) {
+    return bets.reduce((acc, bet) => {
+      const userId = bet.userId
+      if (!acc[userId]) {
+        acc[userId] = []
+      }
+      acc[userId].push(bet)
+      return acc
+    }, {} as { [userId: string]: Bet[] })
+  }
+  const betsByUserId = mapBetsByUserId(bets)
```

Comment:
One suggestion is to use `_.groupBy(bets, bet => bet.userId)`. It's a bit easier!